papaparse.js

```diff
@@ -397,15 +396,15 @@
 					finished: finishedIncludingPreview
 				});
 			}
-			else if (isFunction(this._config.chunk))
+			else if (isFunction(this._config.chunk) && !isFakeChunk)
 			{
 				this._config.chunk(results, this._handle);
-				if (this._paused)
+				if (this._handle.paused())
 					return;
 				results = undefined;
 			}
 
-			if (finishedIncludingPreview && isFunction(this._config.complete) && (!results || !results.meta.aborted))
+			if (finishedIncludingPreview && isFunction(this._config.complete))
```

Comment:
Does this duplicate the complete callback? We call it in the abort function too.